### DISCUSSION TITLE
fichier moonraker.

### DISCUSSION THREAD

**Discussion by yan3700** [bassamanator/Sovol-SV06-firmware#192] [General]
bonjour
j'ai pris un des dossier moonraker.conf(lecture seuil) que j'avez pas avent lecture seuil je les mis dans mon mainsail apres j'ai recuperer sa 
[update_manager printer_config]
type: git_repo
primary_branch: master
path: ~/printer_data/config
origin: https://github.com/bassamanator/Sovol-SV06-firmware.git
managed_services: klipper

est depuis je peux plus l'enlever est je peux plus modifier mais fichier est plus sauvegarder je vous fais une capture d'ecran 
![Capture](https://github.com/user-attachments/assets/26dc84e5-d02a-4aae-944f-0def8b11557d)

**Comment by bassamanator** [comment]
Je ne comprend pas se quoi la problem. :/

**Comment by bassamanator** [comment]
Regardless, from what i see in the picture, this is a bad idea.

**Comment by yan3700** [comment]
depuis que j'ai rajouter c'est lignes de la lignes44 a la ligne 49 j'arrive pas a les supprimer c'est partie la je les prit dans un commentaire ou il parler des mis a jour est depuis je suis en lecture seuil comme indiquer au dessus a gauche j'arrive plus a modifier tous les fichier depuis

**Comment by bassamanator** [comment]
So you're unabled to edit `moonraker.conf` anymore?

What I would do is to edit `moonraker.conf` in the terminal.
1. SSH into the device.
2. `cd ~/printer_data/config`
3. `nano moonraker.conf` (or use `vi` to edit)

Furthermore, unless you have made very intricate changes to the configuration, why not just start from scratch, meaning, delete the contents of `printer_data/confg` and begin again.

**Comment by yan3700** [comment]
justement je voulais repartir de zéro avec une Raspberry Pi 4 que j'ai commander mais comment fais vue qui y'a dejas cette installation sur le pc est sur ubuntu y'a aussi les fichier je c'est pas comment faire

**Comment by bassamanator** [comment]
1. Flash raspberry pi os onto a microSD card
2. Install KIAUH
3. ...

https://www.youtube.com/watch?v=jGplZnNoT4I

**Comment by yan3700** [comment]
Ok merciest sa va remettre après le boîtier klipper en état c'est à dire tous les fonctionnements que j'avais aux débuts ![ Image](https://github.com/user-attachments/assets/726af977-9680-422d-9e8a-b4d8f1cf1295)


J’ai celui  là c’est klipper ou klipperscean que je doit installer 
![image](https://github.com/user-attachments/assets/1b614d7b-71a9-4b96-850d-d8bf596d50da)

**Comment by yan3700** [comment]
Après j’ai tous mais fichier d’origine que j’ai garder je fais comment pour les mettre à la place se que sont sur mainsail quand je l’aurais installer avec le pi

**Comment by bassamanator** [comment]
To reset/update Sovol klipper screen, find instructions here: https://github.com/3DPrintDemon/How-to-Update-Sovol-Klipper-Screen-To-Latest-Klipper-SV06-and-SV07

To install my configuration to the Sovol screen is more complicated. It would take me about 2 minutes, but if you don't know linux, then you'll have a hard time.

Don't forget to star my repo. Thanks.

**Comment by yan3700** [comment]
C’est quoi le lien au dessus fait installer et ou ?

**Comment by yan3700** [comment]
Après je suis peux être pas obligé de reset le boîtier klipper il peux se mettre quand j’aurai installé tous est ajouter l’imprimante dans mainsail

**Comment by yan3700** [comment]
j'ai fais une installation propre avec raspberry j'ai une erreur que j'arrive pas a enlever 
![Capture](https://github.com/user-attachments/assets/fc226cb4-4f04-4a6e-afe6-9b5e9b7c8970)
est pour l'écran il les toujours comme sa depuis la nouvelle installation je suis perdu la  

https://private-user-images.githubusercontent.com/187410033/386874520-726af977-9680-422d-9e8a-b4d8f1cf1295.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4ODY4NTEsIm5iZiI6MTczMTg4NjU1MSwicGF0aCI6Ii8xODc0MTAwMzMvMzg2ODc0NTIwLTcyNmFmOTc3LTk2ODAtNDIyZC05ZThhLWI0ZDhmMWNmMTI5NS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QyMzM1NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MTVmMDY1YmQ4MmJiNDllY2VkMzNjMDBhZGQ2MTRlMDRmMjZlYTVkYmYxODhjZDY0YmI5ZmU0YmU4ODc0ZDlkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._n5XP5ztaO-knn4jzU3fgbN4gZUSoGep8LKvINhC_sk

**Comment by yan3700** [comment]
bonjour changer d'endroit

**Comment by yan3700** [comment]
J’ai suivis votre vidéo YouTube tous c’est bien passer mais sa toutee les commande on etait comme vous mais a la fin vous ça fonctionne mais moi nn 
![image](https://github.com/user-attachments/assets/42db1f84-5458-4ca6-96fc-458bf95aaf80)
